notebooks/CS_VQE usage.ipynb

```diff
@@ -93,6 +93,14 @@
     "A noncontextual/contextual partition is identied:"
```

Comment:
<p>Nice, easier to understand</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UCL-CCS/symmer/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>